marathon.go

```diff
@@ -23,6 +23,10 @@ type portDefinition struct {
 	Name string `json:"name" yaml:"name"`
 }
 
+type checkCommand struct {
```

Comment:
FWIW this could have been an embedded struct (like `HealthChecks` is for `marathonApp`).